ros2topic/test/test_cli.py

```diff
@@ -504,6 +504,7 @@ def test_truncate_length_topic_echo(self):
             ), timeout=10)
         assert topic_command.wait_for_shutdown(timeout=10)
 
+    @launch_testing.markers.retry_on_failure(times=5)
```

Comment:
Why are these changes needed and how are they related to this PR?

Reply:
Those changes slipped in, my bad. These tests flaked me once locally, and later I forgot to take decorators out. We can split into their own PR if you'd like. Truth is I cannot consistently reproduce the flake.

Reply:
Yeah, splitting sounds reasonable since nobody will expect this change in the squashed commit after the fact.